Remove wasm-gc installation prerequisite (#217)
radicle-dev/radicle-registry: DEVELOPING.md

<!-- toc -->

- [Developer Manual](#developer-manual)
    - [Table of contents](#table-of-contents)
  - [Build requirements](#build-requirements)
  - [Running development node](#running-development-node)
  - [Packages](#packages)
  - [Running checks and tests](#running-checks-and-tests)
  - [Changelog](#changelog)
  - [Continuous Deployment](#continuous-deployment)
  - [Local devnet](#local-devnet)
  - [Updating substrate](#updating-substrate)
  - [Updating Continuous Integration's base Docker image](#updating-continuous-integrations-base-docker-image)
  - [Git Flow](#git-flow)
    - [Branches](#branches)

>     - [Commits](#commits)

Reviewer: Could you please revert these changes? There were no changes to the toc.

Updating Continuous Integration's base Docker image
---------------------------------------------------

1. In `.buildkite/pipeline.yaml`, in value of `.test` -> `env` -> `DOCKER_IMAGE` replace image tag (last part after `:`) with a nonexistent tag (e.g. `does_not_exist`).

Example:
```
DOCKER_IMAGE: gcr.io/opensourcecoin/radicle-registry/ci-base:0d7ce69abca7dfe7dcbf26e319645405f31f4901
```
to
```
DOCKER_IMAGE: gcr.io/opensourcecoin/radicle-registry/ci-base:does_not_exist
```

> 2. Push the commit to the repository. **Make sure that this commit ID is preserved!** Do not amend, squash, rebase or delete it, it should be merged unmodified into master. This way it will be easy to look up the state of the project used by the CI.

Reviewer: ```suggestion
2. Push the commit to the repository. **Make sure that this commit is preserved!** Do not amend, squash, rebase or delete it, it should be merged unmodified into master. This way it will be easy to look up the state of the project used by the CI.
```

> **What happens on CI server:** no artifacts can be found for given tag and they must be rebuilt. CI will run the pipeline and save both docker image and its artifacts under tag same as current commit ID.

Reviewer: Some terminology suggestions: build-agent instead of CI server and docker image instead of artifacts.
```suggestion
**What happens on the build agent:** no docker image can be found for the given tag the agent will build the image. The agent will run the pipeline and save the docker image a under tag same as current commit ID.
```
